src/dbnode/integration/options.go

```diff
@@ -171,6 +174,12 @@ type TestOptions interface {
 	// TruncateRequestTimeout returns the truncate request timeout.
 	TruncateRequestTimeout() time.Duration
 
+	// SetFetchRequestTimeout sets the fetch request timeout.
+	SetFetchRequestTimeout(value time.Duration) TestOptions
+
+	// FetchRequestTimeout returns the fetch request timeout.
+	FetchRequestTimeout() time.Duration
```

Comment:
Where is fetch request timeout?

Reply:
It sometimes happens on those `FetchTagged` requests that we run in parallel (4x) with the aggregation. Seen it timing out both on my machine and in CI.

Reply:
(I mean, I increased the timeout to fix the flakiness that I saw on `TestAggregationAndQueryingAtHighConcurrency`)

Reply:
Ah I see, ok sg.